src/components/LoginForm/LoginForm.tsx

```diff
@@ -25,16 +25,16 @@ export default function LoginForm() {
     validationSchema: yup.object({
       email: yup
         .string()
-        .email('Must be a valid email')
-        .min(5, 'Min 5 symbols')
-        .max(50, 'Max 50 symbols')
-        .required('Required'),
+        .email('validation.valid')
+        .min(5, 'validation.min5')
+        .max(50, 'validation.max50')
+        .required('validation.required'),
       password: yup
         .string()
-        .min(8, 'Min 8 symbols')
-        .max(40, 'Max 40 symbols')
-        .matches(/[A-z0-9]/, 'Password should be letters and numbers')
-        .required('Required'),
+        .min(8, 'validation.min8')
+        .max(40, 'validation.max40')
```

Comment:
I m not sure, but max 40 it can inform hacker about how max symbols in password?

Reply:
Ok